.github/workflows/zulip-ci.yml

```diff
@@ -9,6 +9,7 @@ defaults:
 jobs:
   focal_bionic:
```

Comment:
Should we rename this job to something that better expresses it's the backend/frontend test suites?

Reply:
I think coming up with something nice for this is hard. Maybe `focal_backend_bionic_backend_frontend` or just `backend_frontend_test`? And then add a comment on top explaining what this does.

Reply:
I’m of the opinion that we don’t need to cram all of our documentation into a name. Also this is something that we’d type when using the [`act`](https://github.com/nektos/act) command line client. How about just `tests`?

Reply:
Also note that you can separately [control](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idname) the name that’s displayed in the GitHub UI.

Reply:
In that case, I like `unit_tests`. And we already take advantage of the `name` field to display the OS and test type already so that already takes care of expressing what it does.